src/main/webapp/javascript/user-page.js

```diff
@@ -85,18 +87,39 @@ async function loadTypeRecipes(type) {
         addToListButton.style.display = "none";
         removeFromListButton.style.display = "inline-block";
       }
+      // sets up the icon of the other add to list button to a checkmark if it is contained
+      changeIcons(recipesDiv, other);
     }
   }
 }
 
+function changeIcons(recipesDiv, type) {
+    console.log("changeIcons");
+  // TODO: check if they're logged in
+  const recipeIDs = recipesDiv.getElementsByClassName("recipe-id");
+  const addToListButtons = recipesDiv.getElementsByClassName("add-to-" + type + "-btn");
+  const removeFromListButtons = recipesDiv.getElementsByClassName("remove-from-" + type + "-btn");
```

Comment:
It doesn't look like you're using `removeFromListButtons`?

Reply:
remnant from the copy paste
I'll delete it